po/de.po

```diff
@@ -411,12 +400,12 @@ msgstr ""
 #: src/env_universal_common.cpp:1134
 #, c-format
 msgid "Unable to make a pipe for universal variables using '%ls': %s"
-msgstr ""
+msgstr "Kann keine Pipe für die globalen Variablen über '%ls' anlegen: %s"
 
 #: src/env_universal_common.cpp:1142
 #, c-format
 msgid "Unable to open a pipe for universal variables using '%ls': %s"
-msgstr ""
+msgstr "Kann keine Pipe für die globalen Variablen über '%ls' öffnen: %s"
```

Comment:
"Pipe" is probably fine, some other translations also call it "Weiterleitung". My personal favourite here is of course consistency with the (alleged) glibc translation for `EPIPE`, "Rohrbruch".

Reply:
I think "Weiterleitung" adds some ambiguity because it might be an actual pipe or even a symlink and at least everyone i know actually refers to it in German as "die Pipe"